### PR TITLE
Default to new doc+software license (closes #505)

### LINKED ISSUE #505: Software + doc license
A question came up on the mailing list about ReSpec's license... right now, it's pointing to:

 * http://www.w3.org/Consortium/Legal/copyright-documents

But we now have the software + doc license: 

 * http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document

Should Respec default to new license?

### PR DESCRIPTION
**Pull request by marcoscaceres**
This allow an editor to select the W3C's software and document license.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@tobie or @halindrome, r?

File: js/w3c/headers.js [hunk 1]
```diff
@@ -247,30 +250,35 @@ define(
...
-            ,   "w3c-software": {
-                    name:   "W3C Software Notice and License"
-                ,   short:  "W3C"
-                ,   url:    "http://www.w3.org/Consortium/Legal/2002/copyright-software-20021231"
-                }
-            ,   "cc-by": {
-                    name:   "Creative Commons Attribution 4.0 International Public License"
-                ,   short:  "CC-BY"
-                ,   url:    "http://creativecommons.org/licenses/by/4.0/legalcode"
+                cc0: {
```
Comment (marcoscaceres) [inline js/w3c/headers.js]: I can't cope with the other coma style. I'm dyslexic.

**Comment by halindrome** [comment]
LGTM.  I am not certain if the rel="license" will generate the correct RDFa, but I will test it once it is available.

**Comment by marcoscaceres** [comment]
> I am not certain if the rel="license" will generate the correct RDFa

Can you explain a bit more here? Why would additional markup be generated in RDFa?

**Comment by plehegar** [comment]
lgtm (but I don't know about the rel-license bits)

**Comment by marcoscaceres** [comment]
Released in 3.2.75.

**Comment by marcoscaceres** [comment]
Restoring branch temporarily to allow Travis to run successfully.

**Pull request merged**